packages/web-react/src/components/Header/HeaderDialogCloseButton.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useStyleProps } from '../../hooks';
+import { ClickEvent, HeaderDialogCloseButtonProps } from '../../types';
+import { HEADER_DIALOG_CLOSE_BUTTON_LABEL_DEFAULT } from './const';
+import { useHeaderModernStyleProps } from './useHeaderStyleProps';
+import { useHeaderDialogContext } from './HeaderDialogContext';
+import { Icon } from '../Icon';
+
+const HeaderDialogCloseButton = (props: HeaderDialogCloseButtonProps) => {
+  const { label = HEADER_DIALOG_CLOSE_BUTTON_LABEL_DEFAULT, onClick, ...restProps } = props;
```

Comment:
Same as with the `menu` constant and i18n.

Reply:
But this is how we use it in all other components that have a label. It should not be mandatory.

Reply:
Yep. Consider resolved.